nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/HandleHttpRequest.java

```diff
@@ -446,6 +447,7 @@ private SslContextFactory createSslFactory(final SSLContextService sslService, f
 
         sslFactory.setNeedClientAuth(needClientAuth);
         sslFactory.setWantClientAuth(wantClientAuth);
+        sslFactory.setProtocol(sslService.getSslAlgorithm());
```

Comment:
I didn't run this one yet, but does it have the same issue as the ListenHTTP with starting up successfullly, but not supporting the ssl/tls choice when a request is made?